independent-publisher-connectors/SHRTCODE/readme.md

```diff
@@ -0,0 +1,44 @@
+# SHRTCODE
+the privacy-friendly URL Shortener. With this free Link Shortener connector you can make Links shorter and easier to remember.
+
+## Publisher
```

Comment:
@sekharmalla this heading should look like this: `Publisher: Publisher's Name`. You can refer to [readme.md example](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md?plain=1). All of the required headings should be present.